src/needleman_wunch.jl

```diff
@@ -5,11 +5,15 @@
 
 Your documentation here.
 """
-function nwscore(base1::Char, base2::Char) = 1
-    
+function nwscore(base1::Char, base2::Char; match=1, mismatch=-1, gap=-1) 
+    if base1 == base 2
+      return match
```

Comment:
Looks like only a 2 space tab here, though you use 4 spaces elsewhere. Again, totally functional, just aesthetically harder to read.